CONTRIBUTING.md

```diff
@@ -0,0 +1,19 @@
+# Contributing to Nexus Operator
+
+## Have found a bug or have a feature request?
+
+Please, open an issue for us. That will really help us to improve and also others that are using our project. There are templates ready for bug reporting and feature request to make things easier for you.
```

Comment:
```suggestion
Please, open an issue for us. That will really help us improving the Operator and it would benefit other users such as yourself. There are templates ready for bug reporting and feature request to make things easier for you.
```
Wdyt?